### ISSUE TITLE
Reasons to bring back ignore arguments

### ISSUE THREAD

**Issue by SeeingBlue** [AHAAAAAAA/PokemonGo-Map#1608]
It's great that users can now set their own ignore and notification list, but I propose the return of command line arguments for at least ignore so I can force set certain settings across all my devices.

Thanks.

**Comment by andrewda** [comment]
:+1: bringing back some of the removed arguments will also allow ports of PokemonGo-Map to have more control.

**Comment by Darkrift411** [comment]
I agree. First, the GUI is only allowing so many ignore/notify items to be added. Second, I do not like having to fill out my defaults each time I open it in a new browser.
I want to be able to set a global list of ignores (pidgeys, etc) and a global list of ones to notify me on. There are some pokemon that I will likely ALWAYS want to be notified of.

**Comment by abighairyspider** [comment]
I just noticed they added an exclude link to the popups when you select a mon. This makes the process at least tolerable on a new device now. I'd still like to see a global ignore come back...

**Comment by Darkrift411** [comment]
I just thought about this some more, and what if there were some preset ignores in a configuration file. The user could use the defaults, or add new ones. These would then display as checkboxes near the exclude section where you coudl check them and add each pokemon in that preset group to the ignore or notify list.

Example: Trash (in my area) would include - Pidgey,Rattata,Ekans,Sandshrew,Zubat,Paras,Venonat,Mankey,Abra,Geodude,Doduo

**Comment by aprotopapas** [comment]
Global ignore was a great feature. All good ideas, but please bring it back even if it is a command line argument as before. Thanks
